pom.xml

```diff
@@ -46,7 +48,7 @@
 		<dependency>
 			<groupId>org.springdoc</groupId>
 			<artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
-			<version>2.0.2</version>
+			<version>${swagger.api}</version>
 		</dependency>
```

Comment:
change version variable name to openapi.version immediately!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! 👎 👎 👎

Reply:
done 😎